### DISCUSSION TITLE
9.0.1-preview.2

### DISCUSSION THREAD

**Discussion by holytshirt** [BrighterCommand/Brighter#1724] [General]
## Changes

- Fixes #1700 | Adds feature switch support for builders @DevJonny (#1701)
- Make RawMessageDelivery option configurable @honkuan86 (#1648)
- Added Static and Async Methods to Sweeper @preardon (#1707)
- Issue #1634 | Adds support for async handlers when using RPC @DevJonny (#1685)
- Split Bus App Lifetime Dependencies from CommandProcessor Instance Lifetime Dependencies @iancooper (#1628)
- Removed Throw from the Acknowledge on ASB Comsumer @preardon (#1618)
- Updated Outstanding Command SQl Query @preardon (#1594)
- Added Check to Outbox Sweeper to only clear if messages exist @preardon (#1611)
- Update Brighter Builder to allow DI of outbox as well as UseExternalMessaging @preardon (#1601)
- Updated References from System.Data to Microsoft.Data @preardon (#1593)
- Removed All references to Raven DB from Help #1198 @preardon (#1527)
- Update to TaskCompletionSource @holytshirt (#1572)

## 🚀 Features

- Example using Web API of Outbox (contains implementation for Sqlite and MySql) @iancooper (#1708)

## 🐛 Bug Fixes

- Outbox scoped provider @preardon (#1687)
- Outbox scoped provider @preardon (#1667)
- Update SQL Boxes to use Microsoft.Data.SqlClient @preardon (#1584)

## 🛠️ Dependencies

- chore(deps): bump Microsoft.Data.Sqlite from 5.0.9 to 5.0.10 @dependabot (#1722)
- chore(deps): bump AWSSDK.SQS from 3.7.1.9 to 3.7.1.11 @dependabot (#1721)
- chore(deps): bump Microsoft.EntityFrameworkCore from 5.0.9 to 5.0.10 @dependabot (#1720)
- chore(deps): bump MessagePack from 2.3.75 to 2.3.85 @dependabot (#1719)
- chore(deps): bump Npgsql from 5.0.7 to 5.0.10 @dependabot (#1718)
- chore(deps): bump AWSSDK.SecurityToken from 3.7.1.54 to 3.7.1.58 @dependabot (#1717)
- chore(deps): bump Microsoft.EntityFrameworkCore.SqlServer from 5.0.9 to 5.0.10 @dependabot (#1716)
- chore(deps): bump System.Data.SqlClient from 4.8.2 to 4.8.3 @dependabot (#1715)
- chore(deps): bump Microsoft.EntityFrameworkCore.Design from 5.0.9 to 5.0.10 @dependabot (#1714)
- chore(deps): bump AWSSDK.DynamoDBv2 from 3.7.0.65 to 3.7.0.69 @dependabot (#1713)
- chore(deps): bump Confluent.Kafka from 1.7.0 to 1.8.1 @dependabot (#1712)
- chore(deps): bump AWSSDK.SimpleNotificationService from 3.7.2.35 to 3.7.2.39 @dependabot (#1711)
- chore(deps): bump AWSSDK.SQS from 3.7.0.54 to 3.7.1.9 @dependabot (#1709)
- chore(deps): bump FakeItEasy from 7.1.0 to 7.2.0 @dependabot (#1694)
- chore(deps): bump MySqlConnector from 1.3.11 to 1.3.12 @dependabot (#1706)
- chore(deps): bump AWSSDK.SecurityToken from 3.7.1.45 to 3.7.1.54 @dependabot (#1705)
- chore(deps): bump AWSSDK.SQS from 3.7.0.54 to 3.7.1.7 @dependabot (#1704)
- chore(deps): bump AWSSDK.DynamoDBv2 from 3.7.0.54 to 3.7.0.65 @dependabot (#1703)
- chore(deps): bump AWSSDK.SimpleNotificationService from 3.7.2.24 to 3.7.2.35 @dependabot (#1702)
- chore(deps): bump AWSSDK.SecurityToken from 3.7.1.41 to 3.7.1.45 @dependabot (#1688)
- chore(deps): bump FluentAssertions from 6.0.0 to 6.1.0 @dependabot (#1684)
- chore(deps): bump ServiceStack.Redis.Core from 5.11.0 to 5.12.0 @dependabot (#1683)
- chore(deps): bump AWSSDK.SimpleNotificationService from 3.7.2.20 to 3.7.2.24 @dependabot (#1681)
- chore(deps): bump AWSSDK.SQS from 3.7.0.50 to 3.7.0.54 @dependabot (#1680)
- chore(deps): bump AWSSDK.DynamoDBv2 from 3.7.0.50 to 3.7.0.54 @dependabot (#1679)
- chore(deps): bump Microsoft.EntityFrameworkCore.SqlServer from 5.0.8 to 5.0.9 @dependabot (#1676)
- chore(deps): bump Microsoft.Data.Sqlite from 5.0.8 to 5.0.9 @dependabot (#1675)
- chore(deps): bump AWSSDK.DynamoDBv2 from 3.7.0.50 to 3.7.0.52 @dependabot (#1674)
- chore(deps): bump Microsoft.EntityFrameworkCore.Design from 5.0.8 to 5.0.9 @dependabot (#1673)
- chore(deps): bump FluentAssertions from 5.10.3 to 6.0.0 @dependabot (#1672)
- chore(deps): bump AWSSDK.SecurityToken from 3.7.1.39 to 3.7.1.41 @dependabot (#1671)
- chore(deps): bump Microsoft.EntityFrameworkCore from 5.0.8 to 5.0.9 @dependabot (#1670)
- chore(deps): bump Microsoft.NET.Test.Sdk from 16.10.0 to 16.11.0 @dependabot (#1669)
- chore(deps): bump Azure.Identity from 1.4.0 to 1.4.1 @dependabot (#1666)
- chore(deps): bump AWSSDK.SecurityToken from 3.7.1.35 to 3.7.1.39 @dependabot (#1665)
- chore(deps): bump AWSSDK.DynamoDBv2 from 3.7.0.45 to 3.7.0.50 @dependabot (#1664)
- chore(deps): bump AWSSDK.SimpleNotificationService from 3.7.2.16 to 3.7.2.20 @dependabot (#1663)
- chore(deps): bump AWSSDK.SQS from 3.7.0.46 to 3.7.0.50 @dependabot (#1662)
- chore(deps): bump MessagePack from 2.2.113 to 2.3.75 @dependabot (#1660)
- chore(deps): bump AWSSDK.SQS from 3.7.0.46 to 3.7.0.48 @dependabot (#1657)
- chore(deps): bump AWSSDK.SecurityToken from 3.7.1.33 to 3.7.1.35 @dependabot (#1656)
- chore(deps): bump AWSSDK.SimpleNotificationService from 3.7.2.14 to 3.7.2.16 @dependabot (#1655)
- chore(deps): bump AWSSDK.SQS from 3.7.0.45 to 3.7.0.46 @dependabot (#1654)
- chore(deps): bump AWSSDK.SQS from 3.7.0.44 to 3.7.0.45 @dependabot (#1651)
- chore(deps): bump AWSSDK.Extensions.NETCore.Setup from 3.7.0.1 to 3.7.1 @dependabot (#1650)
- chore(deps): bump AWSSDK.DynamoDBv2 from 3.7.0.44 to 3.7.0.45 @dependabot (#1649)
- chore(deps): bump MySqlConnector from 1.3.10 to 1.3.11 @dependabot (#1626)
- chore(deps): bump MessagePack from 2.2.85 to 2.2.113 @dependabot (#1647)
- chore(deps): bump Serilog.Sinks.Console from 3.1.1 to 4.0.0 @dependabot (#1646)
- chore(deps): bump coverlet.collector from 3.0.3 to 3.1.0 @dependabot (#1645)
- chore(deps): bump AWSSDK.SimpleNotificationService from 3.7.2.6 to 3.7.2.14 @dependabot (#1644)
- chore(deps): bump AWSSDK.SQS from 3.7.0.36 to 3.7.0.44 @dependabot (#1643)
- chore(deps): bump AWSSDK.SecurityToken from 3.7.1.24 to 3.7.1.33 @dependabot (#1642)
- chore(deps): bump Microsoft.Data.Sqlite from 5.0.7 to 5.0.8 @dependabot (#1641)
- chore(deps): bump Microsoft.EntityFrameworkCore.Design from 5.0.7 to 5.0.8 @dependabot (#1640)
- chore(deps): bump Microsoft.EntityFrameworkCore.SqlServer from 5.0.7 to 5.0.8 @dependabot (#1639)
- chore(deps): bump AWSSDK.DynamoDBv2 from 3.7.0.36 to 3.7.0.44 @dependabot (#1638)
- chore(deps): bump Microsoft.EntityFrameworkCore from 5.0.7 to 5.0.8 @dependabot (#1637)
- chore(deps): bump Microsoft.Extensions.DependencyInjection from 5.0.1 to 5.0.2 @dependabot (#1636)
- chore(deps): bump RabbitMQ.Client from 6.2.1 to 6.2.2 @dependabot (#1617)
- chore(deps): bump AWSSDK.SecurityToken from 3.7.1.22 to 3.7.1.24 @dependabot (#1616)
- chore(deps): bump AWSSDK.DynamoDBv2 from 3.7.0.33 to 3.7.0.36 @dependabot (#1615)
- chore(deps): bump AWSSDK.SQS from 3.7.0.34 to 3.7.0.36 @dependabot (#1614)
- chore(deps): bump AWSSDK.SimpleNotificationService from 3.7.2.3 to 3.7.2.6 @dependabot (#1613)
- chore(deps): bump AWSSDK.SecurityToken from 3.7.1.21 to 3.7.1.22 @dependabot (#1609)
- chore(deps): bump AWSSDK.SQS from 3.7.0.33 to 3.7.0.34 @dependabot (#1608)
- chore(deps): bump AWSSDK.SQS from 3.7.0.31 to 3.7.0.33 @dependabot (#1607)
- chore(deps): bump AWSSDK.SecurityToken from 3.7.1.19 to 3.7.1.21 @dependabot (#1606)
- chore(deps): bump MySqlConnector from 1.3.9 to 1.3.10 @dependabot (#1605)
- chore(deps): bump AWSSDK.SimpleNotificationService from 3.7.2.1 to 3.7.2.3 @dependabot (#1604)
- chore(deps): bump AWSSDK.DynamoDBv2 from 3.7.0.31 to 3.7.0.33 @dependabot (#1603)
- chore(deps): bump FakeItEasy from 7.0.2 to 7.1.0 @dependabot (#1592)
- chore(deps): bump Npgsql from 5.0.5 to 5.0.7 @dependabot (#1591)
- chore(deps): bump AWSSDK.SQS from 3.7.0.30 to 3.7.0.31 @dependabot (#1590)
- chore(deps): bump AWSSDK.DynamoDBv2 from 3.7.0.30 to 3.7.0.31 @dependabot (#1589)
- chore(deps): bump AWSSDK.SecurityToken from 3.7.1.18 to 3.7.1.19 @dependabot (#1588)
- chore(deps): bump AMQPNetLite from 2.4.2 to 2.4.3 @dependabot (#1587)
- chore(deps): bump AWSSDK.SimpleNotificationService from 3.7.2 to 3.7.2.1 @dependabot (#1586)
- chore(deps): bump Microsoft.Data.Sqlite from 5.0.6 to 5.0.7 @dependabot (#1585)

<hr /><em>This discussion was created from the release <a href='https://github.com/BrighterCommand/Brighter/releases/tag/9.0.1-preview.2'>9.0.1-preview.2</a>.</em>

**Comment by holytshirt** [comment]
@iancooper @honkuan86 @DevJonny  @penderi @preardon

**Comment by iancooper** [comment]
https://twitter.com/BrighterCommand/status/1440629811035009028?s=20

I should really automate getting Brighter to RT releases
